From 585a497b48ebfca1b03b37f739495cae10730693 Mon Sep 17 00:00:00 2001
From: Michael Brown <producer@holotronic.dk>
Date: Wed, 22 Apr 2020 02:36:18 +0200
Subject: [PATCH 3/3] Enable send hostname to dhcp server add some u-boot
 variables

Signed-off-by: Michael Brown <producer@holotronic.dk>
---
 include/configs/platform-top.h | 15 ++++++++++++---
 1 file changed, 12 insertions(+), 3 deletions(-)

diff --git a/include/configs/platform-top.h b/include/configs/platform-top.h
index d0545d48df..dd5cd2fcec 100755
--- a/include/configs/platform-top.h
+++ b/include/configs/platform-top.h
@@ -24,6 +24,7 @@
 #endif
 
 #define CONFIG_CMD_DNS
+#define CONFIG_BOOTP_SEND_HOSTNAME
 
 #define CONFIG_EXTRA_ENV_SETTINGS \
 "autoload=yes\0" \
@@ -31,8 +32,16 @@
 "default_bootcmd=mmcinfo && fatload mmc 0 " \
     "10000000 image.ub && bootm 10000000\0" \
 "4.19tftpboot=setenv bootargs console=ttyPS0,115200 " \
-    "root=/dev/mmcblk0p2 rw earlyprintk rootwait;" \
-    "usb start && dhcp 8000000 design_1_wrapper.bit && " \
+    "root=/dev/mmcblk0p2 rw earlyprintk " \
+    "clk_ignore_unused cpuidle.off=1 rootwait;" \
+    "usb start && dhcp 8000000 holosynthv_wrapper.bit && " \
     "fpga load 0 8000000 $filesize && mmcinfo && " \
     "load mmc 0 10000000 Image && load mmc 0 16000000" \
-    " avnet-ultra96-rev1.dtb && booti 10000000 - 16000000\0"
+    " avnet-ultra96-rev1.dtb && booti 10000000 - 16000000\0" \
+"4.19tftpbootu=setenv bootargs console=ttyPS0,115200 " \
+    "root=/dev/mmcblk0p2 rw earlyprintk clk_ignore_unused " \
+    "cpuclk_ignore_unused cpuidle.off=1 rootwait;" \
+    "usb start && run set_serverip && dhcp 8000000 " \
+    "holosynthv_wrapper.bit && fpga load 0 8000000 " \
+    "$filesize && mmcinfo && fatload mmc 0 10000000 " \
+    "image.ub && bootm 10000000\0"
-- 
2.25.1

